README.md

```diff
@@ -41,11 +42,11 @@ To create a migration, execute `db-migrate create` with a title. `node-db-migrat
 
 ```javascript
 exports.up = function (db, callback) {
-  callback();
```

Comment:
I removed the callback from the docs because that's not what actually gets generated so it was misleading.